src/devices/cpu/i386/i386.cpp

```diff
@@ -3475,6 +3486,14 @@ void i386_device::i386_common_init()
 	m_ferr_handler(0);
 
 	set_icountptr(m_cycles);
+
```

Comment:
There's no reason to create four notifiers, that's just inefficient, they're all going to be called in succession with the same paramter.  Create one, and put the loop in it.